src/QMCWaveFunctions/tests/test_einset.cpp

```diff
@@ -112,9 +112,6 @@ TEST_CASE("Einspline SPO from HDF", "[wavefunction]")
 </tmp> \
 ";
 
-  // monoO
```

Comment:
@markdewing Do we still need monoO? You probably has the best knowledge since you added these lines when this test was created. There are more monoO related lines although we don't have any value checking in the file. If we decide to delete monoO, I can take care of the rest since I'm working on this file to add more spline unit tests.

Reply:
The monoO lines can be removed.